frontend/public/components/_resource-dropdown.scss

```diff
@@ -10,6 +10,9 @@
   .dropdown-menu {
     max-height: 75vh;
     min-width: 290px;
+    @media (min-width: 480px) {
```

Comment:
Nit:  don't interrupt the declarations with the media query.  In this example it's not a big deal as there aren't that many declarations, but when there are many, it can get confusing what declarations belong to what.